bin/test-workflow/7_app_deploy.sh

```diff
@@ -0,0 +1,285 @@
+
+    kubectl delete --ignore-not-found secret dockerpullsecret
+
+    kubectl create secret docker-registry dockerpullsecret \
+      --docker-server=${PULL_DOCKER_REGISTRY_URL} \
+      --docker-username=$DOCKER_USERNAME \
+      --docker-password=$DOCKER_PASSWORD \
+      --docker-email=$DOCKER_EMAIL
+  elif [[ "$PLATFORM" == "openshift" ]]; then
+    announce "Creating image pull secret."
+
+    $cli delete --ignore-not-found secrets dockerpullsecret
+
+    $cli secrets new-dockercfg dockerpullsecret \
+      --docker-server=${PULL_DOCKER_REGISTRY_URL} \
+      --docker-username=_ \
+      --docker-password=$($cli whoami -t) \
+      --docker-email=_
+
+    $cli secrets add serviceaccount/default secrets/dockerpullsecret --for=pull
+  fi
+}
+
+###########################
+init_connection_specs() {
+  test_sidecar_app_docker_image=$(platform_image_for_pull test-sidecar-app)
+  test_init_app_docker_image=$(platform_image_for_pull test-init-app)
+
+  if [[ "$LOCAL_AUTHENTICATOR" == "true" ]]; then
+    authenticator_client_image=$(platform_image_for_pull conjur-authn-k8s-client)
+    secretless_image=$(platform_image_for_pull secretless-broker)
+  else
+    authenticator_client_image="cyberark/conjur-authn-k8s-client"
+    secretless_image="cyberark/secretless-broker"
+  fi
+
+  if [[ "$CONJUR_OSS_HELM_INSTALLED" == "true" ]]; then
+    conjur_appliance_url=${CONJUR_APPLIANCE_URL:-https://conjur-oss.$CONJUR_NAMESPACE.svc.cluster.local}
+  else
+    conjur_follower_name=${CONJUR_FOLLOWER_NAME:-conjur-follower}
+    conjur_appliance_url=https://$conjur_follower_name.$CONJUR_NAMESPACE.svc.cluster.local/api
+  fi
+  conjur_authenticator_url="$conjur_appliance_url/authn-k8s/$URLENCODED_AUTHN_ID"
+}
+
+###########################
+deploy_app_backend() {
+  $cli delete --ignore-not-found \
+     service/test-summon-init-app-backend \
+     service/test-summon-sidecar-app-backend \
+     service/test-secretless-app-backend \
+     statefulset/summon-init-pg \
+     statefulset/secretless-pg \
+     statefulset/summon-init-mysql \
+     statefulset/summon-sidecar-mysql \
+     statefulset/secretless-mysql \
+     secret/test-app-backend-certs
+
+  ensure_env_database
+  case "${TEST_APP_DATABASE}" in
+  postgres)
+    echo "Create secrets for test app backend"
+    $cli --namespace $TEST_APP_NAMESPACE_NAME \
+      create secret generic \
+      test-app-backend-certs \
+      --from-file=server.crt=./etc/ca.pem \
+      --from-file=server.key=./etc/ca-key.pem
+
+    echo "Deploying test app backend"
+
+    # Install postgresql helm chart
+    if [ "$(helm list -q -n $TEST_APP_NAMESPACE_NAME | grep "^summon-sidecar-app-backend-pg$")" = "summon-sidecar-app-backend-pg" ]; then
+        helm uninstall summon-sidecar-app-backend-pg -n "$TEST_APP_NAMESPACE_NAME"
+    fi
+
+    kubectl delete --ignore-not-found pvc -l app.kubernetes.io/instance=summon-sidecar-app-backend-pg
+
+    helm repo add bitnami https://charts.bitnami.com/bitnami
```

Comment:
After discussion with @doodlesbykumbi it was realized that it may be preferable to install the backend helm chart separately from the sample app since different personas might perform those steps